src/domain/collectibles/collectibles.repository.ts

```diff
@@ -24,10 +21,8 @@ export class CollectiblesRepository implements ICollectiblesRepository {
     trusted?: boolean;
     excludeSpam?: boolean;
   }): Promise<Page<Collectible>> {
-    const page = (await this.balancesApiManager.useExternalApi(args.chainId))
-      ? await this._getCollectiblesFromBalancesApi(args)
-      : await this._getCollectiblesFromTransactionApi(args);
-
+    const api = await this.balancesApiManager.getBalancesApi(args.chainId);
```

Comment:
The same thing I've commented in `BalancesRepository` about the `api` abstraction is applicable here.